src/gis/supported_formats.md

```diff
@@ -0,0 +1,218 @@
+# Supported Formats
+<!-- reference -->
+  
```

Comment:
Maybe add some intro text e.g.

When working with Mergin Maps, you can keep using standard data formats you are used to in QGIS. In general, most of the data formats that QGIS can load, can be also used in <MobileAppName />. There are however some minor differences worth highlighting.